recipes/google-cloud-cpp/2.x/conanfile.py

```diff
@@ -0,0 +1,333 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd, cross_building
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualRunEnv, VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, replace_in_file, rmdir
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.56.0"
+
+
+class GoogleCloudCppConan(ConanFile):
+    name = "google-cloud-cpp"
+    description = "C++ Client Libraries for Google Cloud Services"
+    license = "Apache-2.0"
+    topics = (
+        "google",
+        "cloud",
+        "google-cloud-storage",
+        "google-cloud-platform",
+        "google-cloud-pubsub",
+        "google-cloud-spanner",
+        "google-cloud-bigtable",
+    )
+    homepage = "https://github.com/googleapis/google-cloud-cpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "CMakeToolchain"
```

Comment:
In dangerous to have this and call it in the generator method since they conflict and overwrite 

```suggestion
```

Reply:
Removed